.vscode/eslint-plugin.code-workspace

```diff
@@ -4,7 +4,4 @@
       "path": ".."
     }
   ],
-  "settings": {
-    "prettier.prettierPath": ".yarn/unplugged/prettier-npm-3.1.0-708d6027b1/node_modules/prettier"
```

Comment:
This was removed because we've reverted yarn to not use `pnp` (plug and play), so we leverage `node_modules`.